documentation/content/howtocontribute.rst

```diff
@@ -157,3 +157,42 @@ If you like, you can put extra tags at the end:
 You can also create your own tag or use one that’s in common usage such as
 `Co-authored-by` or `Helped-by`.
 
+About Issues Labels
+-------------------
+We use Github issues and milestones as our tool for managing kw project. We use
+issues for keeping track of different things, as a result, we categorize all of
+them with labels; most of the available labels are self-explanatory. However,
+few of them deserver a brief explanation:
+
+* **done: wait for stable**: In the section *Development Cycle and Branches* we
+  explained how our branch management works, recall that we have an unstable
+  branch that soon or later will be applied to the master branch (stable). This
+  label is used for identifying issues already solved on the unstable, but not
+  landed in the master yet. If you see this label, feel free to test the
+  feature/bugfix in the unstable branch.
+* **new feature request**: If you want to request or see a possible new
+  feature, you should use this label.
+* **good first issue**: If you started to use kw, and after that, you feel
+  determined to contribute to it but you don't know where you could start, this
+  label is the answer to your aspiration.
+* **optimization**: Issues that require optimization in kw.
+* **help wanted**: If you see this label, it means that we need any kind of
+  help.
+* **question**: We map user questions with this label.
+* **test**: Issues related to kw test, any kind of test.
```

Comment:
What about the following? "Issues related to kw's test suite. Any kind of test or even the `run_tests` script itself."